.github/workflows/alpha.yml

```
@@ -76,7 +76,7 @@ jobs:
 
       # Using Xcode 15 as the alpha build uses iOS 17 APIs
     - name: Select Xcode
-      run: sudo xcode-select -s /Applications/Xcode_15.0.1.app/Contents/Developer
+      run: sudo xcode-select -s /Applications/Xcode_$(<.xcode-version)/Contents/Developer
```

@jaceklyp you've missed the .app extension here:
```suggestion
      run: sudo xcode-select -s /Applications/Xcode_$(<.xcode-version).app/Contents/Developer
```

---

I believe that was my mistake, @jaceklyp is innocent 😇 Fixed, thanks!